### DISCUSSION TITLE
"Mark as read on scroll" stopped working

### DISCUSSION THREAD

**Discussion by drtn222** [spacecowboy/Feeder#161] [General]
I think it was the Dec. 17 update of the app, as I noticed after updating the app to that version. Tried clearing cache and data, uninstall and reinstall, toggle the option in settings on and off, changed view (from card to compact or super compact). Nothing.

**Comment by spacecowboy** [comment]
This has already been fixed but not released

**Reply by drtn222** [reply]
Thanks so much.
